src/components/dynamic/form/SelectAll.jsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+// import { usePathname } from 'next/navigation'
+import Checkbox from "../Checkbox";
+
+const SelectAll = ({ toggle, onClick, required, text }) => {
```

Comment:
shouldn't need a separate component for this, should be able to use a single checkbox at the end and map thru the items within the questions component